Networking/Networking/Remote/GoogleListingsAndAdsRemote.swift

```diff
@@ -0,0 +1,27 @@
+import Foundation
+
+/// Interface for remote requests to Google Listings & Ads plugin.
+///
+public protocol GoogleListingsAndAdsRemoteProtocol {
+    /// Check Google ads connection for the given site.
+    ///
+    func checkConnection(for siteID: Int64) async throws -> GoogleAdsConnection
+}
+
+/// Google Listings & Ads: Endpoints
+///
+public final class GoogleListingsAndAdsRemote: Remote, GoogleListingsAndAdsRemoteProtocol {
+
+    public func checkConnection(for siteID: Int64) async throws -> GoogleAdsConnection {
+        let path = Paths.connection
+        let request = JetpackRequest(wooApiVersion: .none, method: .get, siteID: siteID, path: path)
```

Comment:
❓ Should we specify `availableAsRESTRequest` as `true`? 

I am asking based on the `GoogleAdsConnectionMapper` being able to parse both with/without data envelope JSON responses.

Reply:
Great catch! I added support for REST request in a85b3f88b1d43f193afb656407dc573533888b9e.